graphics/wincursor.cpp

```diff
@@ -111,9 +113,6 @@ bool WinCursor::readFromStream(Common::SeekableReadStream &stream) {
 	const bool supportOpacity = g_system->hasFeature(OSystem::kFeatureCursorMask);
 	const bool supportInvert = g_system->hasFeature(OSystem::kFeatureCursorMaskInvert);
 
-	_hotspotX = stream.readUint16LE();
-	_hotspotY = stream.readUint16LE();
-
 	// Check header size
 	if (stream.readUint32LE() != 40)
```

Comment:
Is this check still valid after you removed reading 4 bytes from the stream above?

Reply:
Yes, see comment on the seek->skip replacement.